frontend/ui-pablo/app/components/Atoms/BigNumberInput.tsx

```diff
@@ -37,14 +37,16 @@ export const BigNumberInput: React.FC<BigNumberInputProps> = ({
 
   React.useEffect(() => {
     setter && setter(bignrValue);
-  }, [bignrValue, setter]);
+  // eslint-disable-next-line react-hooks/exhaustive-deps
+  }, [bignrValue]);
 
   React.useEffect(() => {
     if (!(value as BigNumber).eq(bignrValue)) {
       setValue(value as BigNumber);
       setStringValue((value as BigNumber).toFixed());
     }
-  }, [value, setValue, setStringValue, bignrValue]);
+  // eslint-disable-next-line react-hooks/exhaustive-deps
+  }, [value]);
```

Comment:
Why `bignrValue` is not in the dependency list? I think we should avoid using this eslint comment exception and improve the desing of the components instead since it prevents the actual checks which could be important, i.e. when someone is updating this code, they may add adittional dependencies and forget to add them to the list. Eslint won't be checking it because of the exception, which breaks the whole point of this rule, since the exception appear at so many places.

Reply:
@markostojanovic087 i agree, i will create a separate PR that fixes solely this functionality